Engine/source/sfx/sfxDevice.h

```diff
@@ -62,8 +62,7 @@ class SFXDevice
          CAPS_VoiceManagement = BIT( 1 ),    ///< Device manages voices on its own; deactivates virtualization code in SFX system.
          CAPS_Occlusion       = BIT( 2 ),    ///< Device has its own sound occlusion handling (SFXOcclusionManager).
          CAPS_DSPEffects      = BIT( 3 ),    ///< Device implements DSP effects (SFXDSPManager).
-         CAPS_MultiListener   = BIT( 4 ),    ///< Device supports multiple listeners.
-         CAPS_FMODDesigner    = BIT( 5 ),    ///< FMOD Designer support.
+         CAPS_MultiListener   = BIT( 4 )     ///< Device supports multiple listeners.
```

Comment:
Leaving trailing comma is OK to avoid changing unnecessary lines. Usually preferred style for enum these days.